crates/samples/rustfmt.toml

```diff
@@ -0,0 +1 @@
+newline_style = "Unix"
```

Comment:
This may clash with some folks that have Git configured to checkout in non-LF. There doesn't appear to be any real value in explicitly setting this.

Reply:
All of the generated code uses "Unix" newline formatting, so if this is not set it marks a ton of code as "changed" for the purpose of git differentiation.

Reply:
Ah we have `*       text eol=lf` in gitattributes, so this is OK. (Ideally we'd delete both of these but not going to fight that battle today.)

Reply:
I gave up trying to fight git on line endings. Life's too short. 😜